libraries/botbuilder-dialogs-adaptive/tests/adaptiveDialog.test.js

```diff
@@ -0,0 +1,134 @@
+const { ConversationState, MemoryStorage, TestAdapter, useBotState, UserState } = require('botbuilder-core');
+const { DialogManager } = require('botbuilder-dialogs');
+const {
+    AdaptiveDialog,
+    BeginDialog,
+    OnBeginDialog,
+    OnIntent,
+    RegexRecognizer,
+    ReplaceDialog,
+    SendActivity,
+} = require('../lib');
+
+describe('AdaptiveDialog', function () {
+    this.timeout(5000);
```

Comment:
Is this necessary? I've found that setting the timeout value manually in a test is often a code smell.

Reply:
I forgot to remove this...